rpi-convert-stage-5.sh

```diff
@@ -118,12 +129,17 @@ install_files() {
   sudo ln -sf /dev/null ${rootfs_dir}/etc/systemd/system/udisks2.service
 
   # Extract Linux kernel and install to /boot directory on rootfs
-  sudo cp ${boot_dir}/kernel7.img ${rootfs_dir}/boot/zImage
+  sudo cp ${boot_dir}/${kernel_img} ${rootfs_dir}/boot/zImage
 
   # Replace kernel with U-boot and add boot script
   sudo mkdir -p ${rootfs_dir}/uboot
 
-  sudo cp ${bin_dir_pi}/u-boot.bin ${boot_dir}/kernel7.img
+  if [ "${device_type}" = "raspberrypi3" ]; then
```

Comment:
`==` instead of `=` right?

Reply:
@apodogrocki yes sure sorry overlooked that -> pushed new version